### PR TITLE
Add test for term search

### LINKED ISSUE #11301: Add test for term searches
Tests are needed for term searches in ElasticSearch

### PR DESCRIPTION
**Pull request by chiatt**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Unit test
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change
Adds a unit test for testing terms

### Issues Solved

Closes #11301

### Checklist

-   I targeted one of these branches:
    - [x] dev/7.6.x (under development): features, bugfixes not covered below
    - [ ] dev/7.5.x (main support): regressions, crashing bugs, security issues, major bugs in new features
    - [ ] dev/6.2.x (extended support): major security issues, data loss issues
-   [ ] I added a changelog in arches/releases
-   [ ] I submitted a PR to arches-docs (if appropriate)
-   [ ] Unit tests pass locally with my changes
-   [ ] I added tests that prove my fix is effective or that my feature works
-   [ ] My test fails on the target branch

### CODE REVIEW

File: arches/app/models/migrations/0005_4_0_1.py [hunk 1]
```diff
@@ -17,12 +17,6 @@ def forwards_func(apps, schema_editor):
     # We get the model from the versioned app registry;
     # if we directly import it, it'll be the wrong version
 
-    # index base Arches concept
```
Comment (aarongundel) [inline arches/app/models/migrations/0005_4_0_1.py:20]: @chiatt are these migration changes related to this PR or something else?

Comment (chiatt) [reply arches/app/models/migrations/0005_4_0_1.py:20]: They are actually related. Those calls to index concepts in migrations end up creating an old, invalid concept mapping which persists once the test database is created and migrated. Errors will occur if a term search is performed without first deleting the invalid concept mapping and creating a new one. I could have replaced the concept index in my test, but I wanted to avoid that because the reason for deleting and recreating the concept index would have been unclear to others.  So, it made sense to get rid of the invalid mapping in the migrations. That way, too, they won't haunt anyone else going forward.

**Review by aarongundel — APPROVED** [review]
LGTM
